src/components/auth/components/AuthLayout.jsx

```diff
@@ -1,25 +1,41 @@
 import React from 'react';
 import Grid from '@material-ui/core/Grid';
 import logo from '../../../assets/img/logo.png';
-import { siteData } from '../constants';
+import { cssClasses, siteData } from '../constants';
 
+// Deconstructing css lasses
+const {
+    AUTH_LAYOUT,
+    AUTH_LAYOUT_BRAND,
+    AUTH_LAYOUT_BRAND_NAME,
+    AUTH_LAYOUT_BRAND_DESC,
+    AUTH_LAYOUT_IMG,
+    LEFT,
+    RIGHT,
+} = cssClasses;
```

Comment:
css classes do not need to be saved in constants js file, use the class names directly in the component